cloudnet-modules/cloudnet-syncproxy/src/main/java/de/dytanic/cloudnet/ext/syncproxy/SyncProxyConstants.java

```diff
@@ -9,6 +11,17 @@
             SIGN_CHANNEL_SYNC_ID_GET_SYNC_PROXY_CONFIGURATION_PROPERTY = "sync_bungee_get_sync_bungee_configuration",
             SYNC_PROXY_SERVICE_INFO_SNAPSHOT_ONLINE_COUNT = "Online-Count";
 
+    public static IPermissionManagement PERMISSION_MANAGEMENT;
+
+    static {
+        try {
+            PERMISSION_MANAGEMENT = (IPermissionManagement) Class.forName("de.dytanic.cloudnet.ext.cloudperms.CloudPermissionsPermissionManagement")
+                    .getDeclaredMethod("getInstance")
+                    .invoke(null);
+        } catch (Exception e) {
```

Comment:
Please only catch the exceptions the can be thrown here. And rename the instance to "ignored"